server/util/qps/qps_test.go

```diff
@@ -0,0 +1,71 @@
+package qps_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/qps"
+	"github.com/stretchr/testify/require"
+)
+
+func TestQPS(t *testing.T) {
+	ticker := make(chan time.Time, 1)
+	counter := qps.NewCounterForTesting(5*time.Second, ticker)
```

Comment:
probably want to `defer counter.Stop()` here

Reply:
done